prow/git/git.go

```diff
@@ -71,6 +71,18 @@ func NewClient() (*Client, error) {
 	}, nil
 }
 
+// NewLocalClient returns a client that uses the given path as its remote
+// endpoint instead of GitHub. The client will clone from remote/org/repo.
+// Repo objects spun out of the client will also hit this path.
+func NewLocalClient(remote string) (*Client, error) {
```

Comment:
This feels more like a SetRemote method on the Client. The argument is not guaranteed somehow to set only local clients?